admin/bin/qserv-configure.py

```diff
@@ -321,8 +324,8 @@ def run(self):
                 # used in templates targets comments
                 config['qserv']['meta_config_file'] = self._meta_config_file
 
-            except ConfigParser.NoOptionError as exc:
-                _LOG.fatal("Missing option in meta-configuration file: %s", exc)
+            except ConfigParser.NoOptionError as exception:
+                _LOG.fatal("Missing option in meta-configuration file: %s", exception)
```

Comment:
"exception" is still "exc" in other places of the code. We should either change them all to "exception" or leave it as "exc."